### DISCUSSION TITLE
border color adjustment for calculated node content

### DISCUSSION THREAD

**Discussion by nnako** [freeplane/freeplane#1520] [Help] [answered]
Hi,

it is a good thing to show within a mindmap which nodes carry calculated (formula) content. And thus distinguish optically if a node contains "original" content or content originating somewhere else ("referencing"). Within the mindmap, referencing content is always identified by a `green border` around the node's content.

Now, there are cases where I don't want to optically show any details of this kind. I want to "pretend" that every node is an "original" node within the mindmap. So even nodes with calculated content show without green border.

Is there a way to achieve this "**hide formula borders**" feature via configuration / customization within the Freeplane editor? Or would this require a deeper dive into the code base?

**Comment by Illegit** [comment, marked as answer]
![image](https://github.com/freeplane/freeplane/assets/62674157/29d56ad3-f2dd-4acc-80f8-2e6e0ac65b25)
please have a look at the following commands in view menu and preferences --> highligth formula... are these answering to your needs?
